assets/src/blocks/Timeline/TimelineEditorScript.js

```diff
@@ -3,20 +3,20 @@ import {
   PanelBody,
   SelectControl,
   CheckboxControl,
-  Tooltip
+  Tooltip,
 } from '@wordpress/components';
-import { InspectorControls } from '@wordpress/block-editor';
-
-import { URLInput } from '../../components/URLInput/URLInput';
-import { useScript } from '../../components/useScript/useScript';
-import { useStyleSheet } from '../../components/useStyleSheet/useStyleSheet';
-import { Timeline } from './Timeline';
-import { languages } from './TimelineLanguages'
-import { URLDescriptionHelp } from './URLDescriptionHelp'
-import { debounce } from 'lodash';
-
-const { RichText } = wp.blockEditor;
-const { __ } = wp.i18n;
+import {InspectorControls} from '@wordpress/block-editor';
+
+import {URLInput} from '../../components/URLInput/URLInput';
+import {useScript} from '../../components/useScript/useScript';
+import {useStyleSheet} from '../../components/useStyleSheet/useStyleSheet';
+import {Timeline} from './Timeline';
+import {languages} from './TimelineLanguages';
+import {URLDescriptionHelp} from './URLDescriptionHelp';
+import {debounce, _} from 'lodash';
```

Comment:
I know I'm the one who told you `_` comes from lodash, but there's actually a console error now:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'noConflict')
```
so I don't know where `_` is supposed to come from 😬

Reply:
Maybe we are supposed to write `import {debounce, noConflict} from 'lodash'` and use it directly, without `_.` 🤔

Reply:
@lithrel can definitely help us out here, he wrote the code 😅

Reply:
I've tested it and `import {debounce, noConflict} from 'lodash'` works 🙂

Reply:
No need for the underscore(`_`)?

Reply:
Nope!